html_xblock/html.py

```diff
@@ -0,0 +1,181 @@
+"""This XBlock will help creating a secure and easy-to-use HTML blocks in edx-platform."""
+
+import logging
+import pkg_resources
+from xblock.core import XBlock
+from xblock.fields import Scope, String
+from xblock.fragment import Fragment
+from xblockutils.resources import ResourceLoader
+from xblockutils.studio_editable import StudioEditableXBlockMixin, loader
+
+from .bleaching import SanitizedText
+from .utils import _
+
+log = logging.getLogger('XBlock.HTML')  # pylint: disable=invalid-name
```

Comment:
Sure, but I put it like this because I saw it's used almost the same way in HTML XModule [here](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/html_module.py#L28).